docs/tutorial/openstack.md

```diff
@@ -56,31 +62,25 @@ openstack volume type list
 kops create cluster \
   --cloud openstack \
   --name my-cluster.k8s.local \
-  --state swift://my-cluster \
+  --state ${KOPS_STATE_STORE} \
   --zones nova \
   --network-cidr 10.0.0.0/24 \
-  --image CentOS \
+  --image <imagename> \
   --master-count=3 \
   --node-count=1 \
-  --node-size 2vCPUx8GB \
-  --master-size 2vCPUx8GB \
-  --etcd-storage-type CBS \
-   --api-loadbalancer-type public \
+  --node-size <flavorname> \
+  --master-size <flavorname> \
+  --etcd-storage-type <volumetype> \
+  --api-loadbalancer-type public \
```

Comment:
I think the external network now **must** be specified.

```
  --os-ext-net tenant-floatingip-net
```
We might also document this add:
```
  --os-kubelet-ignore-az=true
```